CactbotOverlay/CactbotEventSource.cs

```diff
@@ -799,21 +666,8 @@ private class NotifyState {
       public bool? in_game_combat;
       public bool dead = false;
       public string zone_name = null;
+      public JObject jobData = new JObject();
```

Comment:
a tiny nit, but this should be `job_data` and not `jobData` to fit with the surrounding style.